### PR TITLE
Add 'mac-address.conf'

### PR DESCRIPTION
**Pull request by LukeShortCloud**
to disable the random generation of MAC addresses. This causes issues when connecting to some Wi-Fi routers.

### CODE REVIEW

**Comment by LukeShortCloud** [comment]
After this is merged, I need to (1) tag a new release in this `playtron-os-files` repository and (2) update the related RPM to include this new file and build using the new tag.

I'm testing this out on one of my devices right now to ensure this new configuration file and location works as expected. If it does, I'll see in the logs for NetworkManager that my MAC address is consistent when scanning for and connecting to Wi-Fi networks. Someone on the team has always confirmed that the configuration being added to `/etc/NetworkManager/NetworkManager.conf` worked.

**Comment by LukeShortCloud** [comment]
It appears to be working from my testing. My real MAC address is staying the same.

File: usr/lib/NetworkManager/conf.d/mac-address.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[device]
```
Comment (alkazar) [inline usr/lib/NetworkManager/conf.d/mac-address.conf:1]: Should this file have a priority? Perhaps higher than what is already in the folder. I see the following on a clean install of PlaytronOS:
 - 20-connectivity-fedora.conf
 - 22-wifi-mac-addr.conf
 
The contents of `22-wifi-mac-addr.conf` seems to indicate it is responsible for the generated MAC addresses.
 
Maybe this file could be something like `50-playtron.conf`

**Review by alkazar — CHANGES_REQUESTED** [review]

File: usr/lib/NetworkManager/conf.d/mac-address.conf (hunk 1 above)
Comment (LukeShortCloud) [reply usr/lib/NetworkManager/conf.d/mac-address.conf:1]: Good catch! Let me change that.

**Review by alkazar — APPROVED** [review]

**Pull request merged**